### PR TITLE
test: integration test

### PR DESCRIPTION
**Pull request by pmakani**

### CODE REVIEW

File: google-cloud-trace/src/test/java/com/google/cloud/trace/v1/it/ITSystemTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  @AfterClass
+  public static void afterTest() {
+    client.close();
+  }
+
+  @Test
+  public void getTraceTest() throws InterruptedException {
+    Thread.sleep(5000);
```
Comment (rahulKQL) [inline google-cloud-trace/src/test/java/com/google/cloud/trace/v1/it/ITSystemTest.java:84]: Any particular reason for this `Thread.sleep()`?

File: google-cloud-trace/src/test/java/com/google/cloud/trace/v2/it/ITSystemTest.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import com.google.protobuf.Timestamp;
+import java.io.IOException;
+import java.util.UUID;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.threeten.bp.Instant;
+
+public class ITSystemTest {
+  private static TraceServiceClient client;
```
Comment (rahulKQL) [inline google-cloud-trace/src/test/java/com/google/cloud/trace/v2/it/ITSystemTest.java:35]: Give one line space here

**Review by rahulKQL — APPROVED** [review]

File: google-cloud-trace/src/test/java/com/google/cloud/trace/v1/it/ITSystemTest.java (hunk 1 above)
Comment (pmakani) [reply google-cloud-trace/src/test/java/com/google/cloud/trace/v1/it/ITSystemTest.java:84]: yes if i don't use sleep than Not Found

File: google-cloud-trace/src/test/java/com/google/cloud/trace/v2/it/ITSystemTest.java (hunk 2 above)
Comment (pmakani) [reply google-cloud-trace/src/test/java/com/google/cloud/trace/v2/it/ITSystemTest.java:35]: Done

**Pull request closed without merging**